en_ewt-ud-test.conllu

```diff
@@ -23448,7 +23448,7 @@
 # text = Fudruckers.
 1-2	Fudruckers	_	_	_	_	_	_	_	SpaceAfter=No
 1	Fudrucker	Fudrucker	PROPN	NNP	Number=Sing	0	root	0:root	_
-2	s	's	PART	POS	Typo=Yes	1	case	1:case	_
```

Comment:
[Fuddruckers](https://en.wikipedia.org/wiki/Fuddruckers) is actually not possessive, but is spelled with two d's.